src/pages/profile_seeker/Preferences.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+    <p>Wähle aus welche Tiere du suchen möchtest</p>
+    <div class="container">
+        <div class="row g-2">
+            <div v-for="animal of animals" class="col-6">
+                <div class="card" role="button" :class="{ 'bg-primary': animal.actice }"
+                    @click="animal.actice = !animal.actice">
```

Comment:
```suggestion
                    @click="animal.active = !animal.active">
```
Bitte überall anpassen!